wp-mail-validator.php

```diff
@@ -108,6 +108,7 @@ function getmxrr($hostName, &$mxHosts, &$mxPreference)
 function wp_mail_validator_init() {
     add_filter('pre_comment_approved', 'wp_mail_validator_validate_comment_mail', 99, 2);
     add_filter('registration_errors', 'wp_mail_validator_validate_registration_mail', 99, 3);
+    add_filter('woocommerce_registration_errors', 'wp_mail_validator_validate_registration_mail', 10, 3);

```

Comment:
And this is already covered in my previous PR for adding verification to the WooCommerce registration forms.

Reply:
And thank you for this Plugin - made my life quite a bit better :D